charts/super-agent-deployment/values.yaml

```diff
@@ -0,0 +1,110 @@
+  tag: ""  # Defaults to chart's appVersion
+  imagePullPolicy: IfNotPresent
+  # -- The secrets that are needed to pull images from a custom registry.
+  pullSecrets: []
+  # - name: regsecret
+
+# -- Volumes to mount in the containers
+extraVolumes: []
+# -- Defines where to mount volumes specified with `extraVolumes`
+extraVolumeMounts: []
+
+# -- Settings controlling ServiceAccount creation.
+# @default -- See `values.yaml`
+serviceAccount:
+  # -- Whether the chart should automatically create the ServiceAccount objects required to run.
+  create: true
+  annotations: {}
+  # If not set and create is true, a name is generated using the full name template
+  name: ""
+
+# -- Additional labels for chart objects. Can be configured also with `global.labels`
+labels: {}
+# -- Annotations to be added to all pods created by the integration.
+podAnnotations: {}
+# -- Additional labels for chart pods. Can be configured also with `global.podLabels`
+podLabels: {}
+
+# -- Service that points to the super agent.
+# @default -- See `values.yaml`
+service:
+  type: ClusterIP
+  port: 80
+
+# -- Resource limits to be added to all pods created by the integration.
+# @default -- `{}`
+resources: {}
+
+# -- Sets pod's priorityClassName. Can be configured also with `global.priorityClassName`
+priorityClassName: ""
+# -- (bool) Sets pod's hostNetwork. Can be configured also with `global.hostNetwork`
+# @default -- `false`
+hostNetwork:
+# -- Sets security context (at pod level). Can be configured also with `global.podSecurityContext`
+podSecurityContext: {}
+# -- Sets security context (at container level). Can be configured also with `global.containerSecurityContext`
+containerSecurityContext: {}
+
+# -- Sets pod's dnsConfig. Can be configured also with `global.dnsConfig`
+dnsConfig: {}
+
+# Settings controlling RBAC objects creation.
+rbac:
+  # -- Whether the chart should automatically create the RBAC objects required to run.
+  create: true
+
+# -- Sets pod/node affinities. Can be configured also with `global.affinity`
+affinity: {}
+# -- Sets pod's node selector. Can be configured also with `global.nodeSelector`
+nodeSelector: {}
+# -- Sets pod's tolerations to node taints. Can be configured also with `global.tolerations`
+tolerations: []
+
+# -- Adds extra attributes to the cluster and all the metrics emitted to the backend. Can be configured also with `global.customAttributes`
+customAttributes: {}
+
+# -- Configures the integration to send all HTTP/HTTPS request through the proxy in that URL. The URL should have a standard format like `https://user:password@hostname:port`. Can be configured also with `global.proxy`
+proxy: ""
+
+# -- (bool) Send the metrics to the staging backend. Requires a valid staging license key. Can be configured also with `global.nrStaging`
+# @default -- `false`
+nrStaging:
+
+fedramp:
+  # -- (bool) Enables FedRAMP. Can be configured also with `global.fedramp.enabled`
+  # @default -- `false`
+  enabled:
```

Comment:
I would add it in the future when implemented

Reply:
I would prefer, as a first iteration, to have marked everything that comes from the `common-library` as TODO to not forget that we are supporting that at all New Relic's charts. We can do a `grep TODO -R` at any point and see what is missing.

I added TODOs on [#1b027c0](https://github.com/newrelic/helm-charts/pull/1155/commits/1b027c0848604a02513c49424606eacb7d4e03ce)

Let me know what you think, I have no strong opinion.

Reply:
I am fine with the TODOs 😄 Before releasing we can decide to address them or remove them

I would just remove what we know that we will never support 

@gsanchezgavier we could add it to the list of leftovers

Reply:
Already marked as TODO: https://github.com/newrelic/helm-charts/pull/1155/files#diff-93b1805c025a55ca2f62fa3f3feaf426afb9b715fa6e8e2c63ab07c76a6da5afR87

I can mark this as solved :D